content/influxdb/cloud/reference/api/influxdb-1x/_index.md

```diff
@@ -41,7 +45,18 @@ Authorization: Token <token>
 Authorization: Token mYSuP3rs3cREtT0k3N
 ```
 
-##### InfluxQL support
+### Basic authentication
+Basic authentication requires the following credentials:
+
+- **username**: arbitrary username
```

Comment:
even though we technically don't care, maybe we should just ask them to use their cloud2 username here?